include/zephyr/drivers/bluetooth/hci_driver.h

```diff
@@ -139,6 +139,14 @@ enum bt_hci_driver_bus {
 	BT_HCI_DRIVER_BUS_IPM           = 9,
 };
 
+#if defined(CONFIG_BT_HCI_SETUP) || defined(__DOXYGEN__)
```

Comment:
This can in some cases result in an empty struct. 
It compiles on my GCC, but might not on other toolchains.

Reply:
Any ideas how best to fix this? Add a dummy field, or always include `public_addr`?

Reply:
I would vote for always include until someone complains